### ISSUE TITLE
Problem reading Arabic characters from Excel2007

### ISSUE THREAD

**Issue by JefferyHus** [PHPOffice/PHPExcel#1007]
I am using this package inside Fuelphp things works just fine with the composer loader, the Fuelphp rendering configurations are fine also as you can control the output encoding, my proble is when I upload and read an ".xlsx" file that contains arabic characters, the words are not well encoded, e.g:
وﻓﺎرة becomes Ùˆï»“ïºŽØ±Ø©, I have tried to check the Http headers and everything seems to be good, I have tried the utf8_encoding() PHP function but nothing happens so I hope someone can help.

**Comment by MarkBaker** [comment]
PHPExcel should be converting all content that it reads from the file to UTF-8 as it reads it; are you displaying it as UTF-8?

**Comment by JefferyHus** [comment]
Yes I display it as UTF-8, I read the file then I display the content inside my view.
